### DISCUSSION TITLE
How to deploy a default config to many clients?

### DISCUSSION THREAD

**Discussion by Andy74h** [Hofknecht/SystemTrayMenu#383] [Q&A] [answered]
The tool is easy to deploy to a larger number of PCs because it is a portable application. But how to pre-configure the application.
Is it possible to create some kind of a default config that is adopted by all logged in users?

**Comment by Hofknecht** [comment, marked as answer]
@Andy74h 
Thanks for your question!

we think you are looking for this option:
![2022-05-25 22_47_06-Window](https://user-images.githubusercontent.com/52528841/170364209-adfda0f5-2cf6-4878-bbd5-b6557ce060a7.png)

After clicking OK/Save, a "user.config" file will be generated at the assembly location.
You can provide the portable version including 'user.config' to have the same configuration on many clients.

**Comment by HariBo6** [comment]
I'm sorry to bring this question up again after 2 years.
There seems to be a writing permission issue on the "assembly location". (Windows Store installation) goes into C:\Program Files\WindowsApps, where it doesn't write the logfile nor the user.config, at least in my case.  

Would be cool to be able to choose a different path for saving these files. 
Other than that, amazing piece of software👍
